server/src/app.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+import { PrismaModule } from './prisma/prisma.module';
+import { UserModule } from './user/user.module';
+
+@Module({
+  imports: [PrismaModule, UserModule],
```

Comment:
Do you need to have PrismaModule as an import if the UserModule already imports it? Still getting used to Nest so not sure if you need both

Reply:
AFAIK it is necessary to import each module when required by another, I don't think they can be passed down. 

From the docs:
![modules](https://docs.nestjs.com/assets/Shared_Module_1.png)